.goreleaser.yml

```diff
@@ -1,18 +1,30 @@
 # This is an example goreleaser.yaml file with some sane defaults.
 # Make sure to check the documentation at http://goreleaser.com
 builds:
-- env:
-  - CGO_ENABLED=0
-  goos:
-  - linux
-  - freebsd
-  - openbsd
-  goarch:
-    - amd64
-    - arm
-    - arm64
-  ldflags:
-  - -s -w -X "github.com/kopia/kopia/repo.BuildVersion={{.Version}}" -X "github.com/kopia/kopia/repo.BuildInfo={{.Commit}}" -X "github.com/kopia/kopia/repo.BuildGitHubRepo={{.Env.GITHUB_REPOSITORY}}"
+  - env:
+    - CGO_ENABLED=0
+    goos:
+    - linux
+    - freebsd
+    - openbsd
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    ldflags:
+    - -s -w -X "github.com/kopia/kopia/repo.BuildVersion={{.Version}}" -X "github.com/kopia/kopia/repo.BuildInfo={{.Commit}}" -X "github.com/kopia/kopia/repo.BuildGitHubRepo={{.Env.GITHUB_REPOSITORY}}"
+  - id: kopia_boring
+    binary: kopia_boring
+    env:
+      - CGO_ENABLED=1
+      - GOEXPERIMENT=boringcrypto
+      - CC=gcc
+      - CXX=g++
+      - GO_EXTLINK_ENABLED=0
+    ldflags:
+      - -s -w -extldflags "-static" -X "github.com/kopia/kopia/repo.BuildVersion={{.Version}}" -X "github.com/kopia/kopia/repo.BuildInfo={{.Commit}}" -X "github.com/kopia/kopia/repo.BuildGitHubRepo={{.Env.GITHUB_REPOSITORY}}"
```

Comment:
remove -s, so we can find the symbols

Reply:
Since this is only for the boring version, it should be OK, since it does not change the regular version.